frontend/src/App.jsx

```diff
@@ -1,25 +1,30 @@
 import { API_URL } from './config';
 import React, { useState, useEffect } from 'react';
-import { BrowserRouter as Router, Route, Switch, Redirect } from 'react-router-dom';
+import {
+  BrowserRouter as Router,
+  Route,
+  Switch,
+  Redirect,
+} from 'react-router-dom';
 import './App.css';
-import Login from './Login';
-import Navbar from './Navbar';
-import NavbarLoggedOut from './NavbarLoggedOut'
-import Profile from './Profile';
-import Markets from './Markets';
-import Polls from './Polls';
-import Notifications from './Notifications';
-import Create from './Create';
-import About from './About';
-import { UserProvider } from './UserContext';
-import MarketDetails from './MarketDetails';
-import User from './User'
-import Footer from './Footer';
+import Login from './components/login/Login';
+import Navbar from './components/navbar/Navbar';
+import NavbarLoggedOut from './components/navbar/NavbarLoggedOut';
+import Profile from './pages/profile/Profile';
+import Markets from './pages/markets/Markets';
+import Polls from './components/polls/Polls';
+import Notifications from './components/notifications/Notifications';
+import Create from './pages/create/Create';
+import About from './pages/about/About';
+import { UserProvider } from './helpers/UserContext';
+import MarketDetails from './components/marketDetails/MarketDetails';
+import User from './pages/user/User';
+import Footer from './components/footer/Footer';
 
 function App() {
   // state variables
   const [backendData, setBackendData] = useState(null);
-  const [isLoggedIn, setIsLoggedIn] = useState(false);
+  const [isLoggedIn, setIsLoggedIn] = useState(true);
```

Comment:
Should't the default state be that the user is not logged in, until they later actually do log in?